src/Module.h

```diff
@@ -47,6 +47,12 @@ class Module {
     std::string name_;
```

Comment:
This is true, but renaming those would require renaming the accessor functions as well.

(IMHO Halide should adopt the trailing underscore style, but that's a separate issue.)